arch/arm/boot/dts/overlays/dt-blob-overlay.dts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Device tree blob overlay for Revolution Pi by KUNBUS
+ *
+ * RevPi Devices
+ */
+
+#include "rpi-dt-blob-overlay.dts"
+
+
+/ {
+	videocore {
+		pins_cm {
+			pin_config {
+				/* For Core SPI0 CLK */
+				pin@p39 { function = "spi"; termination = "no_pulling"; drive_strength_mA = < 8 >; };
+			}; // pin_config
+			pin_defines {
+			}; // pin_defines
+		};
+
+		pins_cm3 {
+			pin_config {
+				/* For Core 3 and Connect SPI0 CLK */
+				pin@p39 { function = "spi"; termination = "no_pulling"; drive_strength_mA = < 8 >; };
+			}; // pin_config
+			pin_defines {
+				/* For redesigned Core and Connect with HATs */
+				pin_define@ID_SDA {
+					type = "internal";
+					number = <0>;
+				};
+				pin_define@ID_SCL {
+					type = "internal";
+					number = <1>;
+				};
+			}; // pin_defines
+		};
+
+		pins_cm4s {
+			pin_config {
+				/* For Core S and Connect S SPI0 CLK */
```

Comment:
as this? "/* For Core S and Connect S set the drive strength for the whole GPIO bank */

Reply:
My suggestion would be:

```
Increase drive strength in order to improve the signal quality (eg. SPI CLK on Core and Connect)
NOTE: This will increase the drive strength for the whole GPIO bank 1 and not only pin 39
```